lib/policy_machine_storage_adapters/active_record/postgresql.rb

```diff
@@ -9,38 +9,66 @@ class Assignment < ::ActiveRecord::Base
       belongs_to :child, class_name: 'PolicyElement', foreign_key: :child_id
 
       def self.transitive_closure?(ancestor, descendant)
-        descendants_of(ancestor).include?(descendant)
+        descendants_of(ancestor.id).include?(descendant)
       end
 
-      def self.descendants_of(element_or_scope)
+      def self.descendants_of(ids)
```

Comment:
It would. It also removed a join because it's currently not collapsed. But sure we can hash that out on a separate PR. Changed it back for now.